### ISSUE TITLE
Radio buttons in overlays do not convey their role/state [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#305]
The form controls that behave as radio buttons (only one option selected at a time) in the “Select your county”, “Gender identity”, “Age range”, “Race”, and “Ethnicity” overlays do not convey a role of radio or their selected state to TalkBack users.

![overlay-List-Age-Range - NRV - Name Role radio btn](https://user-images.githubusercontent.com/7034772/92144789-36bef780-ede5-11ea-8a25-856f11c80964.png)

**Remediation Guidance**

Use the `accessibilityRole` attribute to give the form group container a role of `radiogroup` and each radio button a role of `radio`. Use the `accessibilityState` attribute to convey the `selected` state of the currently selected button.

```javascript
<View accessibilityRole='radiogroup'>
  <TouchableOpacity
    onPress={…}
    accessibilityRole='radio'
    accessibilityState={{selected: true}}
  >
    <Text>0-17</Text>
  </TouchableOpacity>
  <TouchableOpacity
    onPress={…}
    accessibilityRole='radio'
    accessibilityState={{selected: false}}
  >
    <Text>18-34</Text>
  </TouchableOpacity>
  {/* Rest of options */}
</View>
```

**Comment by floridemai** [comment]
@jonnyjames This should be address now.

**Comment by jonnyjames** [comment]
@floridemai I am confirming this issue is fixed. Closing this ticket.
